### ISSUE TITLE
Cannot read property 'documentElement' of undefined

### ISSUE THREAD

**Issue by binarykitchen** [unshiftio/readystate#15]
For robustness when in a browserless environment or worse, when switching between pages in Angular v1 (yuck) the above error can hapen.

Can you make this line more robust?

```
html = doc.documentElement
```

like that

```
html = doc && doc.documentElement
```

thanks!

**Comment by lpinca** [comment]
Wouldn't the same happen here https://github.com/unshiftio/readystate/blob/006051e559244ae98c93175a22edf51a1c7bbb1d/browser.js#L29?

**Comment by binarykitchen** [comment]
yep looks like

**Comment by 3rd-Eden** [comment]
Can you create PR? I have no problem merging these kind of fixes.

**Comment by binarykitchen** [comment]
here u go 
https://github.com/unshiftio/readystate/pull/16

**Comment by 3rd-Eden** [comment]
Published as 0.4.0
